stable/jenkins/templates/deployment.yaml

```diff
@@ -0,0 +1,118 @@
+{{- $component := printf "%s-master" .Release.Name }}
```

Comment:
In `NOTES.txt` you do this: `"component={{ template "fullname" . }}-master"`. Please make it consistent. Actually, I would not even add the fullname to the component name. I'd just go like this:

```yaml
name: {{ template "fullname" . }}-master
labels:
  ...
  component: master
```